### PR TITLE
Add tags to templates

### PR DESCRIPTION
**Pull request by rayashworth**
adding in UserTags and cleaned up a few items

added checking for cluster pools in the main pool namespace and it will not removed if pools still exist.

added .travis.yml to work with ansible linting

### CODE REVIEW

File: ansible/roles/provision_pool/tasks/provision_pool.yml [hunk 1]
```diff
@@ -106,7 +130,7 @@
   when: 'admin_task == "delete" and cloud == "aws"'
   block:
     - name: Check for claims against pool {{ pool_name }}
-      shell: "oc --no-headers=true get clusterclaims -n walt-poolc | tr -s ' ' | cut -f1 -d' ' | grep ^.*$ > /dev/null; echo $?"
+      shell: "oc --no-headers=true get clusterclaims.hive -n walt-poolc | tr -s ' ' | cut -f1 -d' ' | grep ^.*$ > /dev/null; echo $?"
```
Comment (wkrapohl) [inline ansible/roles/provision_pool/tasks/provision_pool.yml:133]: Remove the walt-poolc

**Review by wkrapohl — COMMENTED** [review]
have some comments

File: ansible/roles/provision_pool/tasks/provision_pool.yml (hunk 1 above)
Comment (rayashworth) [reply ansible/roles/provision_pool/tasks/provision_pool.yml]: got it.

**Review by wkrapohl — APPROVED** [review]
LGTM

**Pull request merged**